portality/upgrade.py

```diff
@@ -106,7 +106,7 @@ def do_upgrade(definition, verbose):
             # When we have enough, do some writing
             if len(batch) >= batch_size:
                 total += len(batch)
-                print "writing ", len(batch), "to", tdef.get("type"), ";", total, "of", max
+                print datetime.now(), "writing ", len(batch), "to", tdef.get("type"), ";", total, "of", max
```

Comment:
wow, what a way to format a string! (it was already like that so I'll let it pass)